store/src/java/com/zimbra/cs/imap/ImapHandler.java

```diff
@@ -2295,6 +2269,69 @@ private boolean doLIST(String tag, String referenceName, Set<String> mailboxName
         return true;
     }
 
+    private void accumulateSelectedFolders(Map<ImapPath, ItemId> paths, Set<ImapPath> selected, Set<String> remoteSubscriptions, String owner) throws ServiceException {
```

Comment:
Can we add unit test cases for these methods

Reply:
will check for it